### ISSUE TITLE
Is this  alive? What about a README file?

### ISSUE THREAD

**Issue by timofonic** [RetroShare/LinksCloud#1]
Hello.

Is this project alive?

If so, what about adding a README file?

Kind regards.

**Comment by electron128** [comment]
no, this project is retired. In RS v0.6 we have "Posted" which is much better.
